<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formpelanggan</class>
 <widget class="QWidget" name="Formpelanggan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>321</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="golonganLabel">
      <property name="text">
       <string>Golongan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="golonganComboBox">
      <item>
       <property name="text">
        <string>I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>II</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>III</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IV</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="statusNikahLabel">
      <property name="text">
       <string>Status Nikah</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="statusNikahComboBox">
      <item>
       <property name="text">
        <string>Menikah</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Belum Menikah</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="jumlahAnakLabel">
      <property name="text">
       <string>Jumlah Anak</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="jumlahAnakLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="masaLanggananLabel">
      <property name="text">
       <string>Masa Kerja</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="masaLanggananLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="bayarTotalLabel">
      <property name="text">
       <string>Bayar Total</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="bayarTotalLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="formGroupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="IDLabel">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IDLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaLabel">
      <property name="text">
       <string>Nama</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="telpLabel">
      <property name="text">
       <string>Telp</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="telpLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="alamatLabel">
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="alamatLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablePelanggan">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
